backend/src/app.ts

```diff
@@ -38,14 +41,22 @@ app.use(passport.session());
 authStrategy(passport);
 
 // Set global headers
-app.use('/api',function(req,res,next){
-    res.header("Content-Type" , "application/json" );
-    next(); // http://expressjs.com/guide.html#passing-route control
+app.use('/api', function (req: express.Request, res: express.Response, next: express.NextFunction) {
+  res.header("Content-Type", "application/json");
+  next(); // http://expressjs.com/guide.html#passing-route control
 });
 
 // Routes
 authRouteSetup(app, passport);
-app.use("/api", isLoggedIn, bookingRouter);
+
+app.use("/api", bookingRouter);
```

Comment:
Just curious, what's the benefit of doing the middleware login check in the route vs using it here?

Reply:
`isLoggedIn` seems to be applying to all routes that match /api even if they don't match /api/booking, so I moved it one level down to keep /api/animal publicly accessible